config/initializers/simple_form.rb

```diff
@@ -45,7 +45,7 @@
     # changed input class from field to usa-input
     b.use :input, class: 'block col-12' # usa-input'
     b.use :hint,  wrap_with: { tag: 'div', class: 'usa-form-hint' }
-    b.use :error, wrap_with: { tag: 'div', class: 'usa-error-message margin-top-neg-205' }
+    b.use :error, wrap_with: { tag: 'div', class: 'usa-error-message' }
```

Comment:
@karlaturcios I know you added a class here to solve some error message alignment issues, but on the teams page it actually causes an error message to be out of alignment, so I removed it. If you could show me which different error message on another form this fixed maybe we can figure out what's going on 🤔